src/unix/bsd/apple/mod.rs

```diff
@@ -662,6 +664,10 @@ s_no_extra_traits! {
         __unused1: *mut ::c_void,       //actually a function pointer
         pub sigev_notify_attributes: *mut ::pthread_attr_t
     }
+
+    pub struct processor_cpu_load_info {
+        cpu_ticks: [::c_ulong; CPU_STATE_MAX],
+    }
```

Comment:
Failure:

```
error[E0308]: mismatched types
   --> src/unix/bsd/apple/mod.rs:669:32
    |
669 |         cpu_ticks: [::c_ulong; CPU_STATE_MAX],
    |                                ^^^^^^^^^^^^^ expected `usize`, found `i32`

error: aborting due to previous error
```

Reply:
Fixed it and added a few other things. Let's see if the CI will be happier. :)